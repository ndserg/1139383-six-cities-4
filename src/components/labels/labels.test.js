import React from "react";
import renderer from "react-test-renderer";
import Labels from "./labels";

it(`Render Labels`, () => {
  const tree = renderer
    .create(<Labels
      label={`Premium`}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
