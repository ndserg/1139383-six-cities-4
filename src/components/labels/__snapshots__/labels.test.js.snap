// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Labels 1`] = `
<div
  className="place-card__mark"
>
  <span>
    Premium
  </span>
</div>
`;
